<?xml version="1.0"?>

<launch>

    <arg name="gdb" default="false"/>
    <arg if="$(arg gdb)" name="launch-prefix" value="gdb -ex run --args"/>
    <arg unless="$(arg gdb)" name="launch-prefix" value=""/>

    <node pkg="cb_base_navigation" type="cb_base_navigation_global_planner_interface" name="global_planner_interface" ns="cb_base_navigation" output="screen" launch-prefix="$(arg launch-prefix)">

        <!-- Global Planner Settings -->        
        <rosparam file="$(find cb_base_navigation)/params/global_planner.yaml" command="load" />

        <!-- Global Costmap Settings -->
        <rosparam file="$(find cb_base_navigation)/params/global_costmap.yaml" command="load" ns="global_costmap" />

    </node>

</launch>
